guides/micronaut-jms-aws-sqs/micronaut-jms-aws-sqs.adoc

```diff
@@ -0,0 +1,133 @@
+common:header.adoc[]
+
+common:requirements.adoc[]
+* An https://aws.amazon.com/[AWS account] with:
+** An IAM user with enough permissions to create and manage a queue instances in SQS.
+** The AWS CLI configured to use the IAM user above.
+
+common:aws.adoc[]
+
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+== Create an application
+Let’s create a set of components that will use the Micronaut JMS to integrate with Amazon SQS. This will allow us to send and receive messages from the SQS. The SQS is a reliable, highly-scalable hosted queue for storing messages as they travel between applications or microservices. Amazon SQS moves data between distributed application components and helps you decouple these components.
```

Comment:
Missing link to SQS and use a blockquote for content which is quoting SQS documentation.